<launch>
  <arg name="input_topic" default="/pacmod/can_tx"/>
  <arg name="output_topic" default="/pacmod/parsed_tx/debug"/>
  <arg name="output_accel_cal_topic" default="/pacmod/parsed_tx/accel_cal_rpt"/>
  <arg name="output_brake_cal_topic" default="/pacmod/parsed_tx/brake_cal_rpt"/>
  <arg name="output_steer_cal_topic" default="/pacmod/parsed_tx/steer_cal_rpt"/>
  <arg name="calibration_active" default="false"/>
  <node pkg="as" exec="pacmod_additional_debug_publisher" name="pacmod_additional_debug_publisher" output="screen">
    <param name="use_sim_time" value="$(env AW_ROS2_USE_SIM_TIME false)"/>
    <remap from="input/can_tx" to="$(var input_topic)" />
    <remap from="output/debug" to="$(var output_topic)" />
    <remap from="output/accel_cal_rpt" to="$(var output_accel_cal_topic)" />
    <remap from="output/brake_cal_rpt" to="$(var output_brake_cal_topic)" />
    <remap from="output/steer_cal_rpt" to="$(var output_steer_cal_topic)" />
    <param name="calibration_active" value="$(var calibration_active)" />
  </node>
</launch>
